### PR TITLE
Support feature based cond-expand

### PR DESCRIPTION
**Pull request by higepon**

### CODE REVIEW

File: boot/baselib/base.scm [hunk 1]
```diff
@@ -37,6 +37,11 @@
                (let ((it ,sym)) ,@(cdr cl1))
                (acond ,@(cdr clauses)))))))
 
+;; R7RS features.
+;; TODO(higepon): Add more.
+(define (available-features)
+  '(r6rs r7rs mosh mosh-0.2.8))
```
Comment (okuoku) [inline boot/baselib/base.scm:43]: We shouldn't expose `mosh-0.2.8` since we have not reached any stable state.

**Review by okuoku — CHANGES_REQUESTED** [review]
It seems we cannot bootstrap with RC4. Perhaps we should create RC5 with `available-features` on base.scm then incorporate it into psyntax.

File: boot/baselib/base.scm (hunk 1 above)
Comment (higepon) [reply boot/baselib/base.scm:43]: Thanks for the catch.

**Comment by higepon** [comment]
> It seems we cannot bootstrap with RC4. Perhaps we should create RC5 with `available-features` on base.scm then incorporate it into psyntax.

Yes. In the mean time I put features both in base.scm and library-converter.

**Pull request merged**